.github/workflows/ci-cd.yml

```diff
@@ -62,5 +62,5 @@ jobs:
               env:
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
                   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-              run: npx semantic-release
+              run: npx semantic-release@^19
```

Comment:
Should this be changed to the new action?

Reply:
`semantic-release@^19` is the tool that does the release. The new action is something separate that just validated PR titles. They just have similar sounding names.